### PR TITLE
feat: Add serverless scope sample

### PR DESCRIPTION
**Pull request by grant**
Add a serverless scope sample.

http://b/142060621

```
nox -s "lint(sample='./functions/tips')"
```

### CODE REVIEW

**Comment by grant** [comment]
Should be ready for review.

File: functions/tips/main.py [hunk 1]
```diff
@@ -50,13 +52,52 @@ def function_specific_computation():
...
 
 # Declared at cold-start, but only initialized if/when the function executes
 lazy_global = None
 
+# [START functions_tips_scopes]
+# [START run_tips_global_scope]
+# Global (instance-wide) scope
+# This computation runs at instance cold-start
+instance_var = heavy_computation()
+# [END functions_tips_global_scope]
```
Comment (averikitsch) [inline functions/tips/main.py]: Can you wrap this code in one region tag not two separate chunk?

Comment (averikitsch) [reply functions/tips/main.py]: Also it looks like you put this code in the middle of another region tag.

Comment (grant) [reply functions/tips/main.py]: Yikes. Thanks for catching that.
The latest diff should be cleaner.

**Pull request merged**